clang/test/Driver/clang-sycl-linker-test.cpp

```diff
@@ -1,41 +1,43 @@
 // Tests the clang-sycl-linker tool.
 //
-// Test a simple case without arguments.
+// Test a simple case to link two fat objects.
```

Comment:
The test doesn't use fat objects.
Please, remove this test case. The next test case does the same check. Instead, we should add a test checking the result of the linking user inputs.

Reply:
I had the same thought. Done in the latest commit.

Reply:
Do you mean https://github.com/asudarsa/llvm-project/pull/4/commits/596488cdf625d2f2a84c381f0ddc4bc0968f73c0?
It does something different. It removes wrong test case, and it misses the check for the linker output.

Reply:
I updated the test and also added a new test to test the result